Cargo.lock

```diff
@@ -963,6 +955,15 @@ name = "libc"
 version = "0.2.51"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 
+[[package]]
+name = "libloading"
```

Comment:
This is a new third-party dependency, per https://github.com/mozilla/application-services/blob/master/docs/dependency-management.md please include a comment on the due-diligence applied in selecting it as part of this PR.

Reply:
Added a comment in the top-level post, thanks!